test/utils/creators/peer.js

```diff
@@ -1,32 +1,71 @@
 'use strict'
 
+const pTimes = require('p-times')
+
+const multiaddr = require('multiaddr')
 const PeerId = require('peer-id')
 const PeerInfo = require('peer-info')
 
+const Libp2p = require('../../../src')
 const Peers = require('../../fixtures/peers')
+const defaultOptions = require('../base-options.browser')
 
-async function createPeerInfo (length) {
-  const peers = await Promise.all(
-    Array.from({ length })
-      .map((_, i) => PeerId.create())
-  )
+const listenAddr = multiaddr('/ip4/127.0.0.1/tcp/0')
 
-  return peers.map((peer) => new PeerInfo(peer))
-}
+/**
+ * Create libp2p nodes.
+ * @param {Object} [properties]
+ * @param {Object} [properties.config]
+ * @param {number} [properties.number] number of peers (default: 1).
+ * @param {boolean} [properties.fixture] use fixture for peer-id generation (default: true)
+ * @param {boolean} [properties.started] nodes should start (defaul: true)
+ * @return {Promise<Array<Libp2p>>}
+ */
+async function createPeer ({ number = 1, fixture = true, started = true, config = defaultOptions } = {}) {
+  const peerInfos = await createPeerInfo({ number, fixture })
 
-function createPeerIdsFromFixture (length) {
-  return Promise.all(
-    Array.from({ length })
-      .map((_, i) => PeerId.createFromJSON(Peers[i]))
-  )
+  const peers = await pTimes(number, (i) => Libp2p.create({
+    peerInfo: peerInfos[i],
+    ...config
+  }))
+
+  if (started) {
+    await Promise.all(peers.map((p) => {
+      p.peerInfo.multiaddrs.add(listenAddr)
+      return p.start()
+    }))
+  }
+
+  return peers
 }
 
-async function createPeerInfoFromFixture (length) {
-  const peers = await createPeerIdsFromFixture(length)
+/**
+ * Create Peer-ids.
+ * @param {Object} [properties]
+ * @param {number} [properties.number] number of peers (default: 1).
+ * @param {boolean} [properties.fixture] use fixture for peer-id generation (default: true)
+ * @return {Promise<Array<PeerInfo>>}
+ */
+async function createPeerInfo ({ number = 1, fixture } = {}) {
```

Comment:
`createPeerId` is setting fixture to true, but it might be good to also set the default here since the docs say the default is true. This will prevent any changes to `createPeerId` from affecting this.
```suggestion
async function createPeerInfo ({ number = 1, fixture = true } = {}) {
```